src/main/java/com/microsoft/jenkins/kubernetes/credentials/KubeconfigCredentials.java

```diff
@@ -205,9 +205,10 @@ public String getDisplayName() {
         private transient Item owner;
 
         @DataBoundConstructor
-        public FileOnKubernetesMasterKubeconfigSource(String server, String sshCredentialId) {
+        public FileOnKubernetesMasterKubeconfigSource(String server, String sshCredentialId, String file) {
```

Comment:
File should not be put in the `@DataBoundConstructor` annotation since it is optional.